packages/favorites/__test__/FavoriteHeart.test.js

```diff
@@ -112,25 +118,6 @@ describe('FavoriteHeart', () => {
     await waitForElement(() => getByText('This item is not favorited.'));
   });
 
-  test('should call onChange once toggled', async () => {
```

Comment:
@GoPro16 i had to move this test to the end of the file to get them to pass. I don't know why. I suspect some sort of timing issue with the clean up that runs after each test.